parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -551,7 +551,15 @@ ensureGPUDrivers() {
     fi
 
     if [[ "${CONFIG_GPU_DRIVER_IF_NEEDED}" = true ]]; then
-        logs_to_events "AKS.CSE.ensureGPUDrivers.configGPUDrivers" configGPUDrivers
+        existing_version="$(nvidia-smi | grep "Driver Version" | cut -d' ' -f3)"
+
+        if [[ "*$existing_version*" == $NVIDIA_DRIVER_IMAGE_TAG ]]; then
+            echo "Existing version matches, validating..."
+            logs_to_events "AKS.CSE.ensureGPUDrivers.validateGPUDrivers" validateGPUDrivers
+        else
+            echo "Existing version $existing_version does not match $NVIDIA_DRIVER_IMAGE_TAG"
```

Comment:
I think we need to bump the tag to pull in the image with your changes still, right? https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/cse_helpers.sh#L100

Reply:
looks like sha-9564c5
 from https://github.com/Azure/aks-gpu/actions/runs/3293098403/jobs/5429235929